src/index.html

```diff
@@ -52,23 +57,17 @@ <h2 class="text-center text-md-start mt-5 mb-4">Bringing industry standards to C
     </p>
     <p>
       Supported by the
-      <a
-        rel="noreferrer"
-        target="_blank"
-        class="red-link fw-bold"
-        href="https://calsta.ca.gov/">California State Transportation Agency</a>
+      <a rel="noreferrer" target="_blank" class="fw-bold" href="https://calsta.ca.gov/">California State Transportation Agency</a>
       (CalSTA) and the
-      <a
-        rel="noreferrer"
-        target="_blank"
-        class="green-link fw-bold"
-        href="https://dot.ca.gov/ ">California Department of Transportation</a>
+      <a rel="noreferrer" target="_blank" class="fw-bold" href="https://dot.ca.gov/ ">California Department of Transportation</a>

```

Comment:
I know this isn't part of this PR, but I just noticed this `href` has an extra space at the end, before the end quote.

Reply:
Fixed: https://github.com/cal-itp/calitp.org/pull/361/commits/2b626f1e7c65beb7f4f247b2f016688755080c74